react/features/conference/middleware.ts

```diff
@@ -113,5 +144,117 @@ function _conferenceJoined({ dispatch, getState }: IStore) {
         getState
     });
 
+    if (!intervalId) {
+        intervalId = setInterval(() =>
+            _maybeDisplayCalendarNotification({
+                dispatch,
+                getState
+            }), 10 * 1000);
```

Comment:
Does this mean we'll be updating the notification every 10 seconds? That sounds good.